src/cli-constant.js

```diff
@@ -241,9 +258,19 @@ const detailedOptions = normalizeDetailedOptions({
     category: CATEGORY_FORMAT,
     forwardToApi: true,
     choices: [
-      "none",
-      "es5",
-      "all",
+      { value: "none", description: "No trailing commas." },
+      {
+        value: "es5",
+        description:
+          "Trailing commas where valid in ES5 (objects, arrays, etc.)"
+      },
+      {
+        value: "all",
+        description: dedent(`
+          Trailing commas wherever possible (including function arguments).
+          This requires node 8 or a transform.
```

Comment:
I understand that you copied this from the readme, but I don't think we should include information that will be quickly outdated here. We can change the readme at any time, but changing the CLI description requires a new version to be release. Both Firefox and Chrome supports trailing function argument commas (not sure about other browsers). I think we can leave out the "This requires node 8 or a transform." sentence (and update the readme at some point).

Reply:
👍